### DISCUSSION TITLE
Run different set of commands on different cisco routers with a script

### DISCUSSION THREAD

**Discussion by Cloud7Paradox** [ktbyers/netmiko#3116] [Q&A]
HI all , If anyone can suggest on creating a python script where I should be able to Run different set of commands on different cisco routers  at once as I need to send config commands to around 1200 routers

**Comment by yielditercode** [comment]
celery delay method ; to exec commands

**Reply by Cloud7Paradox** [reply]
The configuration files are separate for each routers ( 1200 configs for 1200 routers)
